lib/canvas.js

```diff
@@ -0,0 +1,140 @@
+"use strict";
```

Comment:
The file name is too generic. It's not a Canvas, it's a... what? It's part of the developer's job to find good names.